source/assets/stylesheets/_accessibility.scss

```diff
@@ -60,6 +60,11 @@ a:focus {
   outline: 3px solid $focus-colour;
 }
 
+/* Make links slightly darker when focused to improve contrast. */
+a:link:focus {
```

Comment:
I'm not proposing changing the visited colour when focused. It will stay as purple - which meets our contrast requirements.